docs/Reference.md

```diff
@@ -1127,6 +1128,18 @@ Example:
 
 `<WebView textZoom={100} />`
 
+### `autoManageStatusBarEnabled`
+
+If the user watch fullscreen video in WebView then StatusBar will be light. The default value is `true`.
```

Comment:
Perhaps 'If the user watches fullscreen video in WebView then StatusBar color will be light.' might be better,